cirq-core/cirq/ion/ion_device.py

```diff
@@ -56,10 +56,18 @@ def __init__(
             oneq_gates_duration: The maximum duration of a single qubit
             operation.
             qubits: Qubits on the device, identified by their x, y location.
```

Comment:
Remove `identified by their x, y location` because it's not by location anymore.

Reply:
nice catch!